rails-commenting.rb

```diff
@@ -6,19 +6,19 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# 1)
+# 1)this is the controller for the app
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) The index is used to view all the entries that are in the database
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3)This method allows the user to look into an entry thats specified
     @post = BlogPost.find(params[:id])
   end
 
-  # 4)
+  # 4)This will take the user to a new page
```

Comment:
def new allows the user to create or post a new instance or form in the database, which will in turn take you to another page if properly setup.